src/main/webapp/partials/instructorHelpSearch.jsp

```diff
@@ -0,0 +1,365 @@
+        <li>
+            <span class="text-bold">
+                <a name="searchStudents">
+                    <h3>1. Searching for students</h3>
+                </a>
+            </span>
+            <div>
+                You can search for students by clicking on
+                <b>'Search'</b> on the top nav bar. You should see a search bar similar to the one below -
+                <br><br>
+                <div class="well well-plain">
+                    <div class="form-group">
+                        <div class="input-group">
+                            <input type="text" name="searchkey"
+                                   title="Search for comment"
+                                   placeholder="Your search keyword"
+                                   class="form-control">
+                            <span class="input-group-btn">
+                                <button class="btn btn-primary" type="submit"
+                                        value="Search">
+                                    Search
+                                </button>
+                            </span>
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <ul class="list-inline">
+                            <li>
+                                <span data-toggle="tooltip" title="Tick the checkboxes to limit your search to certain categories"
+                                      class="glyphicon glyphicon-info-sign">
+                                </span>
+                            </li>
+                            <li>
+                                <input id="comments-for-student-check" type="checkbox">
+                                <label for="comments-for-student-check">
+                                    Comments for students
+                                </label>
+                            </li>
+                            <li>
+                                <input id="comments-for-responses-check" type="checkbox">
+                                <label for="comments-for-responses-check">
+                                    Comments for responses
+                                </label>
+                            </li>
+                            <li>
+                                <input id="students-check" type="checkbox" checked>
+                                <label for="students-check">
+                                    Students
+                                </label>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <br>Check the option <b>'Students'</b> below the search box. Now type the student's name in the search box and click on <b>'Search'</b> button.
+                <br>If you searched <b>'alice'</b>, and let's assume such student exists, then the search results would show something like this -
```

Comment:
`If you searched for 'alice', the search results would show something like this (assuming such a student exists):`